src/block/timeline/edit.js

```diff
@@ -112,6 +135,149 @@ const Edit = props => {
 		'stk-block-content',
 	], getContentAlignmentClasses( props.attributes ) )
 
+	// get previous and next elements/blocks
+	const getAdjacentBlocks = () => {
+		let timelineBlock = document.getElementById( 'block-' + clientId )
+
+		if ( ! timelineBlock ) {
+			const iframe = document.querySelector( 'iframe' )
+			timelineBlock = iframe.contentDocument.getElementById( 'block-' + clientId )
+		}
+
+		const nextBlock = timelineBlock ? timelineBlock.nextElementSibling : null
+		const previousBlock = timelineBlock ? timelineBlock.previousElementSibling : null
+
+		return {
+			previousBlock,
+			nextBlock,
+		}
+	}
+
+	// check if timeline block is single, is first block, or is last block
+	const getTimelinePosition = ( previousBlock, nextBlock ) => {
```

Comment:
Because of the `nextBlock` and `previousBlock` above, you can now just check the `nextBlock.name === 'stackable/timeline'` and `previousBlock.name === 'stackable/timeline'` to check here